ariadne/cardano/src/Ariadne/Config/Wallet.hs

```diff
@@ -34,16 +39,20 @@ parseFieldWallet = parseField walletFieldModifier
 walletFieldModifier :: D.Text -> D.Text
 walletFieldModifier = f
   where
-    f "wcEntropySize"  = "entropy-size"
+    f "wcEntropySize"       = "entropy-size"
     -- name is identical to the one used in Cardano for UserSecret
-    f "wcKeyfilePath" = "keyfile"
-    f "wcAcidDBPath"  = "wallet-db-path"
+    f "wcKeyfilePath"       = "keyfile"
+    f "wcAcidDBPath"        = "wallet-db-path"
+    f "wcNumStoredArchives" = "stored-db-archives"
+    f "wcDBCleanupPerion"   = "db-cleanup-period"
     f x = x
 
 data WalletConfig = WalletConfig
-  { wcEntropySize :: !Byte
-  , wcKeyfilePath :: !FilePath
-  , wcAcidDBPath  :: !FilePath
+  { wcEntropySize        :: !Byte
+  , wcKeyfilePath        :: !FilePath
+  , wcAcidDBPath         :: !FilePath
+  , wcNumStoredArchives  :: !Int
+  , wcDBCleanupPerion    :: !Int
```

Comment:
Why are you using `Int` instead of `o-clock`'s `Second`?

Reply:
Also, there is one extra space for alignment.